sw/device/lib/testing/mock_mmio_test.cc

```diff
@@ -20,18 +20,22 @@ using ::testing::Test;
 uint32_t WriteTwice(mmio_region_t dev) {
   auto value = mmio_region_read32(dev, 0x0);
   mmio_region_write32(dev, 0x4, ~value);
-  mmio_region_write16(dev, 0x8, value >> 16);
-  mmio_region_write16(dev, 0xa, value & 0xffff);
+  mmio_region_write8(dev, 0x12, (value >> 24) & 0xffff);
+  mmio_region_write8(dev, 0x8, (value >> 16) & 0xffff);
+  mmio_region_write8(dev, 0x4, (value >> 8) & 0xffff);
+  mmio_region_write8(dev, 0x0, value & 0xffff);
```

Comment:
Minor: I think these masks should be `0xff` for the sake of consistency and the first one is not really required.

Reply:
This is a good catch, which I should have worked out :)